src/foam/nanos/approval/test/LiquidApprovalCreateRejectTest.java

```diff
@@ -0,0 +1,16 @@
+package foam.nanos.approval.test;
+
+import foam.core.X;
+import foam.nanos.approval.ApprovalStatus;
+import foam.nanos.auth.LifecycleState;
+import foam.nanos.test.Test;
+
+
+// Reject create
+public class LiquidApprovalCreateRejectTest extends ApprovalTestExecutor {
```

Comment:
should liquid specific test be on foam?

Reply:
It was there all the time. Where do you want me to move it?